MapboxNavigation/DefaultStyle.swift

```diff
@@ -3,17 +3,23 @@ import Foundation
 
 extension UIColor {
     class var defaultRouteCasing: UIColor { get { return .defaultTintStroke } }
-    class var defaultRouteLayer: UIColor { get { return UIColor.defaultTintStroke.withAlphaComponent(0.6) } }
+    class var defaultRouteLayer: UIColor { get { return #colorLiteral(red:0.00, green:0.70, blue:0.99, alpha:1.0) } }
```

Comment:
It might be confusing that there is no color for the route layer in the style but `unknown` & `low` is being used there right? However, what's route layer color in case of biking/walking or a style w/o traffic?